src/api_v2/models/user.rs

```diff
@@ -0,0 +1,73 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+use crate::api_v2::Error;
+
+/// ```
+/// if User::uid == 0{
+///     .. //bot user
+/// } else {
+///     .. //user
+/// };
+/// ```
+///
+pub struct User {
+    pub uid: i32,
+    pub time: u64,
+    pub access_token: String,
+    pub refresh_token: String,
+}
+
+pub trait UserSave {
```

Comment:
In Rust naming convention, it will be better to use `*er` for trait object. So here we should use `UserSaver`